build.cmd

```diff
@@ -24,10 +24,10 @@ if not exist %NUGET_LOCAL_DIR%\NuGet.exe (
 )
 
 :: restore packages
-%NUGET_LOCAL_DIR%\NuGet.exe restore .\packages.config -PackagesDirectory ./src/packages -MSBuildVersion 15 -Verbosity quiet
+%NUGET_LOCAL_DIR%\NuGet.exe restore .\packages.config -PackagesDirectory ./build-packages -MSBuildVersion 15 -Verbosity quiet
```

Comment:
I think I'd prefer to call `build-packages` just `packages` if that works. Otherwise, we should probably rename `packages.config` to `build-packages-config`. I guess either strategy could work, although I'm leaning towards the shorter names.